packages/backend/src/ee/services/ai/tools/findFields.ts

```diff
@@ -15,7 +11,9 @@ type Dependencies = {
 };
 
 export const getFindFields = ({ getExplore }: Dependencies) => {
-    const schema = toolFindFieldsArgsSchema;
+    const schema = z.object({
+        exploreName: z.string(),
+    });
```

Comment:
reason for not declaring this in the `schema` file is because this tool will be replaced soon by the `newFindFields` I assume?

Reply:
yeah, but it caused further type errors. I'll be fixing this properly

Reply:
done